style/body/footer/__content/footer__content.css

```diff
@@ -5,10 +5,13 @@
     height: 50px;
 }
 
+.footer__content h2 {
+    font-weight: 400;
+}
+
 .footer__content .text_size_small, .footer__content .link-button {
```

Comment:
According to our Codding Standards, we should write selectors on different lines: https://www.screencast.com/t/RgjKdPSAG